web-client/src/views/Public/TrialSessions/PublicMobileTrialSessionsDataRow.tsx

```diff
@@ -0,0 +1,60 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
These files below are just moved from `TrialsSessions` to `TrialSessions`, along with commenting back in the link to `TrialSessionDetails`.